packages/react/src/animation/Animation.stories.module.css

```diff
@@ -16,11 +16,7 @@
   inset: var(--base-size-16);
   filter: blur(var(--base-size-48));
   transform: translateZ(0);
-  background: linear-gradient(
-    to right,
-    hsl(var(--base-color-scale-blue-2-hsl)),
-    hsl(var(--base-color-scale-purple-2-hsl))
-  );
+  background: linear-gradient(to right, hsl(208 100% 74%), hsl(269 100% 83%));
```

Comment:
Did this one still need to use HSL to work, or can we switch to using the hex base tokens?